mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -14,24 +14,33 @@
 import xarray as xr
 import datetime
 from netCDF4 import Dataset as netcdf_dataset
-import os.path
 
-from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, remove_repeated_time_index
+from ..shared.mpas_xarray.mpas_xarray import preprocess_mpas, \
+    remove_repeated_time_index
 from ..shared.plot.plotting import plot_global_comparison
 from ..shared.interpolation.interpolate import interp_fields, init_tree
 from ..shared.constants import constants
 
 from ..shared.io import StreamsFile
 
-def ocn_modelvsobs(config, field):
+
+def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
 
     """
     Plots a comparison of ACME/MPAS output to SST or MLD observations
 
+    If present, streamMap is a dictionary of MPAS-O stream names that map to
+    their mpas_analysis counterparts.
+
+    If present, variableMap is a dictionary of MPAS-O variable names that map
+    to their mpas_analysis counterparts.
+
     Authors: Luke Van Roekel, Milena Veneziani, Xylar Asay-Davis
-    Modified: 10/27/2016
+    Modified: 12/07/2016
     """
 
+    field = field.lower()
```

Comment:
@pwolfram, a side note.  I think you're reviewing the whole code because I've made a lot of little changes as part of the PEP8 compliance.  But a lot of these things aren't related to this PR so let's be careful that we focus on the job at hand and not the *entire* code.